code/game/objects/items/storage/boxes.dm

```diff
@@ -863,16 +863,77 @@
 	for(var/i in 1 to 6)
 		new /obj/item/ammo_casing/shotgun/beanbag(src)
 
-/obj/item/storage/box/actionfigure
+/obj/item/storage/box/figurines
 	name = "box of action figures"
-	desc = "The latest set of collectable action figures."
+	desc = "The latest set of collectable figurines."
 	icon_state = "box"
 
-/obj/item/storage/box/actionfigure/PopulateContents()
+/obj/item/storage/box/figurines/PopulateContents()
 	for(var/i in 1 to 4)
-		var/randomFigure = pick(subtypesof(/obj/item/toy/figure))
+		var/randomFigure = pick(pick(subtypesof(/obj/item/toy/figure)),
+			pick(subtypesof(/obj/item/toy/prize)),
+			pick((subtypesof(/obj/item/toy/talking) - typesof(/obj/item/toy/talking/codex_gigas))),
+			pick((subtypesof(/obj/item/toy/plush) - typesof(/obj/item/toy/plush/goatplushie/angry/kinggoat))))

```

Comment:
might want to also remove the goat guard and royal goat guard plushies from this loot pool (normal angry goat plushies are okay, though, they can stay in)

should awakened plushies only be available as space loot (or from a select few other sources), or is okay for them be in this loot pool?

Reply:
This is the same code for what shows up in the plush crate that can be ordered from cargo. Do you think that cargo's plush pool should be adjusted as well?

Reply:
yes

Reply:
So currently the goat guard and royal goat guard can be ordered through cargo?